pkg/endpoint/endpoint.go

```diff
@@ -2248,21 +2248,6 @@ func (e *Endpoint) Delete(conf DeleteConfig) []error {
 	return errs
 }
 
-// GetProxyInfoByFields returns the ID, IPv4 address, IPv6 address, labels,
-// SHA of labels, and identity of the endpoint. Returns an error if the endpoint
-// is in the process of being deleted / has been deleted.
-func (e *Endpoint) GetProxyInfoByFields() (uint64, string, string, []string, string, uint64, error) {
```

Comment:
How's this change related to the commit description?

Reply:
Dead code after `LogRecord.localEndpointInfo` was removed. I split this change to a separate commit to make this clearer.